src/lib/y2storage/encryption_processes/base.rb

```diff
@@ -58,9 +61,12 @@ def create_device(blk_device, dm_name)
 
       private
 
-      # Returns the Y2Storage::EncryptionType to be used
+      # Returns the encryption type to be used
+      #
+      # This method must be defined by derived class.
       #
       # @return [Y2Storage::EncryptionType]
+      abstract_method :encryption_type
       def encryption_type
```

Comment:
Did you not removed the previous definition of `#encryption_type`?

Reply:
Sorry so much. Clearly a left over because of rush. Nice catch!